pkg/internal/util/cmd.go

```diff
@@ -30,7 +30,7 @@ import (
 // CmdLogMixin adds necessary CLI flags for logging and setups the controller runtime log
 func CmdLogMixin(cmd *cobra.Command) *cobra.Command {
 	dev := cmd.PersistentFlags().Bool("development", terminal.IsTerminal(int(os.Stdout.Fd())), "format output for console")
-	v := cmd.PersistentFlags().Int8P("verbose", "v", 0, "verbosity level")
+	v := cmd.PersistentFlags().Int8P("verbose", "v", 4, "verbosity level")
```

Comment:
So this set's the default verbosity to level 4 right?
Don't think we want to have debug logging enabled by default.

Reply:
In the syslog, 4 is info level, 6 is the debug level. Thus we should honor the syslog conventions.